README.md

```diff
@@ -26,100 +86,215 @@ A storage abstraction beyond the existing storage services.
 - [oss](https://github.com/beyondstorage/go-service-oss/): [Aliyun Object Storage](https://www.aliyun.com/product/oss)
 - [qingstor](https://github.com/beyondstorage/go-service-qingstor/): [QingStor Object Storage](https://www.qingcloud.com/products/qingstor/)
 - [s3](https://github.com/beyondstorage/go-service-s3/): [Amazon S3](https://aws.amazon.com/s3/)
+
+**1** beta services that implemented required functions, but not passed [integration tests](https://github.com/beyondstorage/go-integration-test).
+
 - [uss](https://github.com/beyondstorage/go-service-uss/): [UPYUN Storage Service](https://www.upyun.com/products/file-storage)
 
-### Servicer operation support
+**11** alpha services that still under development.
+
+- [ftp](https://github.com/beyondstorage/go-service-ftp/): FTP
+- [gdrive](https://github.com/beyondstorage/go-service-gdrive): [Google Drive](https://www.google.com/drive/)
+- [hdfs](https://github.com/beyondstorage/go-service-hdfs): [Hadoop Distributed File System](https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html#Introduction)
+- [ipfs](https://github.com/beyondstorage/go-service-ipfs): [InterPlanetary File System](https://ipfs.io)
+- [memory](https://github.com/beyondstorage/go-service-memory): data that only in memory
+- [minio](https://github.com/beyondstorage/go-service-minio): [MinIO](https://min.io)
+- [onedrive](https://github.com/beyondstorage/go-service-onedrive): [Microsoft OneDrive](https://www.microsoft.com/en-ww/microsoft-365/onedrive/online-cloud-storage)
+- [storj](https://github.com/beyondstorage/go-service-storj): [StorJ](https://www.storj.io/)
+- [tar](https://github.com/beyondstorage/go-service-tar): tar files
+- [webdav](https://github.com/beyondstorage/go-service-webdav): [WebDAV](http://www.webdav.org/)
+- [zip](https://github.com/beyondstorage/go-service-zip): zip files
 
-- List: list all Storager in service
-- Get: get a Storager via name
-- Create: create a Storager
-- Delete: delete a Storager
+More service ideas could be found at [Service Integration Tracking](https://github.com/beyondstorage/go-storage/issues/536).
 
-### Storager operation support
+### Complete and easily expandable interface
 
 Basic operations
 
-- Metadata: get storager metadata
-- Read: read file content
-- Write: write content into file
-- Stat: get file's metadata
-- Delete: delete a file or directory
-- List: list file in prefix or dir styles
+- Metadata: get `Storager` metadata
+```go
+meta := store.Metadata()
+_ := meta.GetWorkDir() // Get object WorkDir
+_, ok := meta.GetWriteSizeMaximum() // Get the maximum size for write operation
+```
```

Comment:
Will the code snippets too wordy? (I'm OK with it though)

Reply:
Btw, whether adding code snippets here, I think we need to add them to the doc later. 

And I'm wondering the relationship between this README and doc homepage https://beyondstorage.io/docs/go-storage/index

Reply:
I think the doc homepage should be redesigned.